<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>25</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="toolTip">
      <string>Press to close the window</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="findtextCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="findPrevButton">
     <property name="toolTip">
      <string>Press to find the previous occurrence</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="findNextButton">
     <property name="toolTip">
      <string>Press to find the next occurrence</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="caseCheckBox">
     <property name="text">
      <string>Match case</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="wordCheckBox">
     <property name="text">
      <string>Whole word</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="regexpCheckBox">
     <property name="text">
      <string>Regexp</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="wrapCheckBox">
     <property name="text">
      <string>Wrap around</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="selectionCheckBox">
     <property name="text">
      <string>Selection only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findtextCombo</tabstop>
  <tabstop>caseCheckBox</tabstop>
  <tabstop>wordCheckBox</tabstop>
  <tabstop>regexpCheckBox</tabstop>
  <tabstop>wrapCheckBox</tabstop>
  <tabstop>selectionCheckBox</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>findPrevButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
